portal-web/test/functional/com/liferay/portalweb/macros/headless/headlessdeliveryapi/CommentAPI.macro

```diff
@@ -43,6 +43,37 @@ definition {
 		return "${curl}";
 	}
 
+	macro _curlBlogPostingChildComments {
+		Variables.assertDefined(parameterList = "${commentId}");
+
+		var portalURL = JSONCompany.getPortalURL();
+
+		var curl = '''
+			${portalURL}/o/headless-delivery/v1.0/comments/${commentId}/comments \
+			-u test@liferay.com:test \
+			-H Content-Type: application/json
+		''';
+
+		return "${curl}";
+	}
+
+	macro _curlBlogPostingChildCommentsByExternalReferenceCode {
+		Variables.assertDefined(parameterList = "${blogPostingChildCommentERC},${blogPostingCommentERC}");
```

Comment:
Should we use uppercase with the first character? I mean blogPostingChildCommentErc and blogPostingCommentErc.